crates/primitives/src/block.rs

```diff
@@ -57,10 +57,16 @@ impl Block {
     ///
     /// # Panics
     ///
-    /// If the number of senders does not match the number of transactions in the block.
+    /// If the number of senders does not match the number of transactions in the block
```

Comment:
does it panic now when `number of senders does not match the number of transactions`?

Reply:
yes, if both conditions are true, it will panic